src/components/snapshot/overview/ManageWorkspacesView.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { ClassNameMap, createStyles, withStyles } from '@mui/styles';
+import { SnapshotWorkspaceEntry } from 'models/workspaceentry';
+import { Chip, CustomTheme } from '@mui/material';
+
+const styles = (theme: CustomTheme) =>
+  createStyles({
+    workspaceChip: {
+      margin: theme.spacing(1),
+    },
+    workspaceChipContainer: {
+      margin: theme.spacing(1),
+      maxHeight: theme.spacing(20),
+      overflowY: 'scroll',
+      width: '100%',
+    },
+  });
+
+type ManageWorkspaceViewProps = {
+  classes: ClassNameMap;
```

Comment:
Can be done in a followup but you can avoid explicitly having to add classes by making the type union with the `WithStyles` tab